### PR TITLE
[Azure AD] Cleaned sample

### PR DESCRIPTION
**Pull request by gal42**
Added readme to match the samples in the dotnet/maui-samples repo

### CODE REVIEW

File: AzureADSample/AzureADSample/Service/Constants.cs [hunk 1]
```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace AzureADSample.Service;
+﻿namespace AzureADSample.Service;
 
 public static class Constants
```
Comment (davidortinau) [inline AzureADSample/AzureADSample/Service/Constants.cs:3]: Think we need to remove the client id string and put a placeholder that the developer replaces with their own ID from Azure. Link here also to docs where they can find it.

File: AzureADSample/README.md [hunk 2]
```diff
@@ -17,4 +30,8 @@ Built in .NET MAUI and used Azure AD for Auth Service. This sample only demonstr
 
 ## Nugget Packages
```
Comment (davidortinau) [inline AzureADSample/README.md:31]:
```suggestion
## NuGet Packages
```

File: AzureADSample/README.md [hunk 3]
```diff
@@ -17,4 +30,8 @@ Built in .NET MAUI and used Azure AD for Auth Service. This sample only demonstr
...
 - `Microsoft.Identity.Client`
-- `System.IdentityModel.Tokens.Jwt`
\ No newline at end of file
+- `System.IdentityModel.Tokens.Jwt`
+
+For more information about the sample see:
```
Comment (davidortinau) [inline AzureADSample/README.md:35]: Add some explanation of what AD stands for, and MSAL.

**Review by davidortinau — COMMENTED** [review]
added a few feedback items.

**Review by tj-devel709 — APPROVED** [review]
I think these changes look great! Other than the comments left by Dave, I think this LGTM (Looks good to me!)

File: AzureADSample/README.md [hunk 4]
```diff
@@ -15,6 +29,17 @@ Built in .NET MAUI and used Azure AD for Auth Service. This sample only demonstr
 </p>
 
 
-## Nugget Packages
+## Nuget Packages
```
Comment (tj-devel709) [inline AzureADSample/README.md:32]:
```suggestion
## NuGet Packages
```

**Pull request merged**